src/Pages/Components/Main.jsx

```diff
@@ -0,0 +1,9 @@
+export default function Main(props) {
+  const { projects } = props;
```

Comment:
This is fine; you can also destruct props directly in the parameter like so:
```
export default function Main({ projects })
```

Reply:
Fixed that. It's a better way =)